app/models/comment.rb

```diff
@@ -0,0 +1,12 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  after_save :update_comments_counter
+
+  private
+
+  def update_comments_counter
+    post.increment!(:comments_counter)
+  end
+end
```

Comment:
Great work here using [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_comments_counter` method every time a new post has been saved to the database. 👏👏